src/main/resources/templates/springboot/application-class.java.tmpl

```diff
@@ -9,8 +9,7 @@ import org.springframework.boot.autoconfigure.security.SecurityAutoConfiguration
 import org.springframework.boot.orm.jpa.EntityScan;
 import org.springframework.context.annotation.ComponentScan;
 
-@SpringBootApplication
-@EntityScan({"${servicePackage}", "com.blackbaud.boot.converters"})
+@SpringBootApplication${entityScan}
```

Comment:
it's a bit odd having ${entityScan} at the end of line and relying on the \n in the entityScan text... consider just putting ${entityScan} on the line above and having an extra space if you don't have the entity scan... the other option would be to generate the basic template and then using FileUtils.appendAfterLine to append the EntityScan line after SpringBootApplication